charts/headlamp/templates/deployment.yaml

```diff
@@ -91,6 +91,9 @@ spec:
             {{- with .Values.config.baseURL }}
             - "-base-url={{ . }}"
             {{- end }}
+            {{- range .Values.config.extraArgs }}
+            - "{{ . }}"
```

Comment:
What about an extra space here or at the end of the base-url part? 

Otherwise it could end up like `"-base-url=example-otherarg"` ?

Reply:
Adding extra space or keeping it like this, did not effect it. I tried it but the output for the helm template was as follows

```yaml
args:
    - "-in-cluster"
    - "-plugins-dir=/headlamp/plugins"
    - "-base-url=/example"
    - "-insecure-ssl"

```